reframe/utility/typecheck.py

```diff
@@ -93,8 +93,9 @@
 
 '''
 
-import abc
+import pdb
 import re
+import abc
```

Comment:
I guess `pdb` is left by mistake. Also `abc` should go where it was (before `re`).